### DISCUSSION TITLE
Cinema LED loses audio and reduces to about 50% brightness after install and one adjustment :(

### DISCUSSION THREAD

**Discussion by tylermitchellbeach** [MonitorControl/MonitorControl#1439] [Q&A]
I have no idea what Im doing just to preface this. I'm bad ass at making epic videos but I know how to flush my caches and respect the PMC on my old Macbook in terminal lol

Someone help haha 

As I said, lost audio, reduced to 50% brightness on 27" Cinema Display, that has a display port to usb c dongle into my M1 Mac Mini. 

I literally just bought this Mac Mini and installed FCPX and all my plugins onto a factory reset unit(fresh version of Sonora 14.0) so that I could work on two videos at once bouncing back and fourth with my new MBP allow for either machine to render and then progress on the other one.. 

The LED display seems to be the only one having issues.  The portable 15.6 KYY Amazon special plugged in via HDMI works as expected.. 

What do I do?

**Comment by Davidino71** [comment]
Same issue here: I have both a thunderbolt display and a LED Cinema Display connected to my MacBook M1 Pro, and after changing the brightness via the app, they're now both stuck at 50% brightness, mind you, Monitor Control still displays sliders at 100%, and I still can dim it down to 0%, but increasing back to 100% both screens still stay dimmed, impossible to get back the full brightness, I uninstalled the app restarted my Mac and both monitors, but the brightness is still low even though control panel shows them both at 100%...

Any Idea anyone ?

**Comment by waydabber** [comment]
Guys, please check for all the relevant threads in Discussions, this question was asked about a 100 times already. Change the DDC range to 0-255 for these displays. :)

**Reply by Davidino71** [reply]
Thank you! Found it here: https://github.com/MonitorControl/MonitorControl/discussions/1381
